### DISCUSSION TITLE
How can I display downvotes?

### DISCUSSION THREAD

**Discussion by sallyFoster** [FreeTubeApp/FreeTube#4473] [Q&A]
I want to be able to see downvotes

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
We don't have a way to extract dislikes  

see the comment here for a full explanation: https://github.com/FreeTubeApp/FreeTube/issues/1927#issuecomment-1007121970

TL;DR the backend is still closed source and they don't offer database dumps, so it is not possible to host your own instance if you don't trust the original one. For comparison the SponsorBlock and DeArrow backends are open source and they provide frequent database dumps.
